modules/migration-understanding-mtc.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc
+// migration/migrating_4_1_4/migrating-application-workloads-4_1-to-4.adoc
+// migration/migrating_4_2_4/migrating-application-workloads-4_2-to-4.adoc
+[id='migration-understanding-mtc_{context}']
+= Understanding the {mtc-full}
+
+The {mtc-first} enables you to migrate Kubernetes resources, persistent volume data, and internal container images from an {product-title} source cluster to an {product-title} {product-version} target cluster, using the {mtc-short} web console or the Kubernetes API.
+
+Migrating an application with the {mtc-short} web console involves the following steps:
+
+. Install the {mtc-operator} on all clusters.
++
+[NOTE]
+====
+The {mtc-operator} installs the {mtc-short} web console and the Migration controller on the target cluster by default. You can configure the Migration controller to install the {mtc-short} on another cluster.
```

Comment:
This note appears in the preview as 

The Cluster Application Migration Operator installs the CAM tool (CAM web console and Migration controller) on the target cluster by default. You can configure the Migration controller to install the CAM tool on another cluster. The cluster on which the CAM tool is installed is referred to as the CAM tool cluster.

Not only are the variables returned oddly, but there is an extra sentence.